### ISSUE TITLE
Saying that extra commas should be added to tsconfig.json

### ISSUE THREAD

**Issue by adjenks** [microsoft/vscode#121522]
- VS Code Version: 1.54.3
- OS Version: Windows

Steps to Reproduce:

1. Create tsconfig.json
2. View it. Perhaps there's something else in my system that makes this unique but I don't know what it is. I have zero extensions installed.

Does this issue occur when all extensions are disabled?: Yes

Asking for 2 extra commas with content:
![image](https://user-images.githubusercontent.com/10687569/115093383-fe30e180-9ece-11eb-8848-8c4daec41a97.png)

Asking for 2 extra commas with zero content:
![image](https://user-images.githubusercontent.com/10687569/115093416-156fcf00-9ecf-11eb-9e1e-8b14f2e4eebf.png)

**Comment by adjenks** [comment]
Restarting VS Code solved the problem for some reason... Random weird stuff like this keeps happening. What would cause this?

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by mjbvz** [comment]
Also, what was the language mode of the file?

**Comment by adjenks** [comment]
At the time, I did not have any extensions installed. I cannot reproduce the error now because it stopped happening.

This is what I did and experienced:
1. I noticed the weird behaviour.
2. I deleted everything except the open and close tags to see if it would still happen, it still happened.
3. I added everything back.
4. I checked the language mode, it was "JSON with Comments"
5. I renamed the file to `somethingelse.json` and it stopped parsing it oddly and appeared as only json format.
6. I renamed it back to `tsconfig.json` and it went back to "JSON with Comments" mode and the error re-appeared.
7. I restarted VS Code and checked the file and I am no longer having the problem.

I have had many several bugs like this though, with syntax highlighting where it just underlines random things and tells me there are problems. It's happened in a variety of file types. Feel free to close the ticket since I cannot reproduce it on command, but just take note that error checking and syntax highlighting has been acting strangely.
